README.md

```diff
@@ -26,18 +27,16 @@ This helm chart installs and configures the following operators:
 
 ### Install
 
-Run `helm upgrade --install <release-name> <path-to-chart>` to deploy default installations of the necessary operators.
+Run `helm upgrade --install <release-name> <path-to-chart> --namespace <namespace> --create-namespace` to deploy default installations of the necessary operators.
 
 #### Example
 
-`helm upgrade --install setup-default-operators ./chart`
+`helm upgrade --install ai-rhdh ./chart --namespace rhdh --create-namespace`
```

Comment:
I think `rhdh` is okay for the namespace since this will just be a normal install of RHDH that is compatible with AI software templates even though it isn't required you use them.

Reply:
I agree with @Jdubrick, I don't think its too important what a user uses here but I will change it still to use consistent naming under the example.

Reply:
Nice, yeah if we're going to stick to the normal installation of RHDH is fine to keep the `rhdh` name